programmers/알고리즘(khw970421)/level2/폰켓몬.js

```diff
@@ -0,0 +1,62 @@
+// 처음 성공 코드
+function solution(nums) {
+    const set1 = new Set(nums);
+    const n_divide_length = nums.length/2;
+    var result = Array.from(set1);
+    if(n_divide_length<result.length){
+        return n_divide_length;
+    }
+    else if(n_divide_length>=result.length){
+        return result.length;
+    }
```

Comment:
```suggestion
    if(n_divide_length<result.length){
        return n_divide_length;
    }
    if(n_divide_length>=result.length){
        return result.length;
    }
```
return 이후에는 else를 사용할 필요 없어요~